var/spack/repos/builtin/packages/hdf5-vol-daos/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class Hdf5VolDaos(CMakePackage):
+    """The HDF5 DAOS VOL connector is an external VOL connector
+    that interfaces with the DAOS API"""
+
+    homepage = "https://github.com/HDFGroup/vol-daos"
+    url = (
+        "https://github.com/HDFGroup/vol-daos/releases/download/v1.2.0/hdf5_vol_daos-1.2.0.tar.bz2"
+    )
+    git = "https://github.com/HDFGroup/vol-daos.git"
+
+    maintainers("hyoklee", "soumagne")
+
+    version("master", branch="master", submodules=True)
+    version("1.2.0", sha256="669c1443605068f24c033783ef72619afcec4844902b3e0bffa19ddeea39779f")
+
+    depends_on("cmake@2.8.12.2:", type="build")
+    depends_on("daos@master")
```

Comment:
you probably want to use a specific version here, not master. Maybe at least `2.0:` I don't think there's any reason for supporting earlier versions of daos so I would also remove all the previous versions in the daos package also and start from 2.0 or even 2.0.3.

Reply:
@soumagne , thank you so much for your feedback!

Reply:
@hyoklee just talked with @mchaarawi it would be actually best if you only start from daos 2.2.0 since daos 2.0 is considered EOL and there won't be any support for it anyway.

Reply:
@soumagne , thank you for contacting M. and providing a valuable guidance. I'll clean up old versions and related packages.